ITMO_FS/filters/unsupervised/trace_ratio_laplacian.py

```diff
@@ -93,12 +90,14 @@ def run(self, X, y):
         e = np.absolute(np.diag(E))
         b = np.absolute(np.diag(B))
         b[b == 0] = 1e-14
-        features_indices = np.argsort(np.divide(b, e))[::-1][0:self.n_selected_features]
-        lam = np.sum(b[features_indices]) / np.sum(e[features_indices])
+        self.selected_features_ = np.argsort(np.divide(b, e))[::-1][0:self.n_features]
+        lam = np.sum(b[self.selected_features_]) / np.sum(e[self.selected_features_])
         prev_lam = 0
         while (lam - prev_lam >= 1e-3):
```

Comment:
Replace that magic number with input value and add todo optimize

Reply:
added